lib/ansible/module_utils/network/common/cfg/base.py

```diff
@@ -13,6 +13,12 @@
 class ConfigBase(object):
     """ The base class for all resource modules
     """
+    ACTION_STATES = ['merged', 'replaced', 'overridden', 'deleted']
```

Comment:
since the PR for the newly introduced states code is merged (ref: #65534), you can skip respective changes in this PR